### ISSUE TITLE
Make it clearer that Redocly CLI is API description format agnostic

### ISSUE THREAD

**Issue by lornajane** [Redocly/redocly-cli#1263]
**Is your feature request related to a problem? Please describe.**

We support linting of a few different structures now, and especially since introducing AsyncAPI support in #1127, we should take care to make sure that the tool doesn't refer specifically to OpenAPI when it's actually any API description format.

**Describe the solution you'd like**

- [x] Change the output of the linting command to not refer to OpenAPI specifically, either to be generic or to pick up and state the file type that was valid
- [x] Review the language of the command parameters and help output to make sure that it is only specific about OpenAPI where we do only have support for thar format
- [ ] Update the documentation for Redocly CLI (yes, all of it) to adjust the wording to be "API description" where it makes sense

**Describe alternatives you've considered**

Hope that nobody notices

**Comment by techstacksavvy** [comment]
Hi @lornajane! 

I'd love to work on this issue. From what I understand, the goal is to replace any instances of the term OpenAPI in the documentation with API description. Is that correct?

**Comment by lornajane** [comment]
It's not a simple find-and-replace (or it would be already done!). The task is to review all the instances where the phrase is used, and check whether it should be made more general or not (not all commands support not-OpenAPI, for example). I'd recommend having some familiarity with the tool before starting this task.

**Comment by techstacksavvy** [comment]
I figured it couldn't be that simple, lol, thanks for clarifying.
